packages/event-migration/src/index.ts

```diff
@@ -37,6 +41,10 @@ const Config = z
       "dev-refactor_attribute_registry",
       "uat-attribute_registry",
       "prod-attribute_registry",
+      "purpose",
+      "dev-refactor_purpose",
+      "uat-purpose",
+      "prod-purpose",
```

Comment:
```suggestion
      "purpose",
      "dev-refactor_purpose",
      "test_purpose",
      "prod_purpose",
```
The <env>_<domain> pattern should be applied to older schemas, but we have already migrated them.

Reply:
Updated